.openshiftio/booster.yaml

```diff
@@ -1,2 +1,7 @@
 name: WildFly Swarm - HTTP
 description: Runs a WildFly Swarm HTTP application
+versions:
```

Comment:
Not sure if we need to get JohnC to ok the user facing names for the versions or not?

Also, raised a question on ML as to whether profiles can ever work as I had forgotten about the openshift profile